### ISSUE TITLE
2D annotation diagonal options

### ISSUE THREAD

**Issue by jrobinso** [aidenlab/juicebox.js#177]
We need the option to display 2D tracks on upper left half, lower right half, or both (entire map), with both as a default.   This is on a per-track basis,  so it will be an extra column in the 2D track edit line  (pull down with 3 options).    The 2D track object should hold the state,   and used in ContactMatrixView when the track is painted.    Not sure what the options should be called, the labels need to be reasonably short.   Perhaps UL, LR, and Full?

**Comment by jrobinso** [comment]
Here's the option menu from Juicebox desktop
<img width="642" alt="screen shot 2017-10-10 at 8 05 45 pm" src="https://user-images.githubusercontent.com/933148/31420477-94cd9232-adf6-11e7-820d-7826ca2d2238.png">

**Comment by sa501428** [comment]
Oh, this is a super outdated version of juicebox...

- Muhammad Saad Shamim <http://mshamim.com/>

**Comment by sa501428** [comment]
For modeling of 2d annotation behavior, it should ideally follow the 2d panel:
<img width="1294" alt="screen shot 2017-10-10 at 10 44 16 pm" src="https://user-images.githubusercontent.com/6503789/31421363-9db2b20e-ae0c-11e7-8bb1-02cb490cf324.png">

**Comment by sa501428** [comment]
Note that the toggle button showing the partial boxes replaced our notation of LL, UR, etc.

**Comment by ibochkov** [comment]
Hi, has this "partial plotting" feature been implemented in the public Juicebox.js? I can't seem to find the toggle button anywhere on the 2D annotations panel.

**Comment by jrobinso** [comment]
You're right, its not there,  I though this had been implemented.   I will
investigate.

**Comment by jrobinso** [comment]
Apparently it was implemented in my imagination only.   I'll put this on the list for the next feature release,  which will be sometime in the fall.

**Comment by turner** [comment]
> *This was generated by AI during triage.*

Closing: **this is implemented.** The 2018 note that it "was implemented in my imagination only" is out of date.

- `Track2D.DisplayModes = { lower, upper, both }` — `js/track2D.js:7`
- The three-state toggle icon (all → lower → upper → all), matching the mock in this thread — `js/annotationWidget.js:137-157` and `displayModeHandler` at `:275`
- Honoured at paint time via `featureDrawPasses(track2D.displayMode, sameChr)` — `js/contactMatrixView.js:742`
- Persisted per-track through `Track2D.toJSON()`

Per-track, three options, short labels — as specified.
